app/assets/javascripts/categories.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
We should remove JS files that scaffold generator generates for us if we do not put any js code in them. they are just poluting the worktree